packages/demo/usecase-demo/src/modules/DisclaimerModal.tsx

```diff
@@ -34,32 +33,36 @@ export function DevModal() {
   useEffect(() => {
     setStorageDevMode(isDevMode);
   }, [isDevMode]);
-  
+
   useEffect(() => {
     if (open == false) {
```

Comment:
Better to never use this `==` syntax.
I propose here:

```
if (open === false) {
```

Reply:
done